### DISCUSSION TITLE
Does this support the Wyze doorbell too?

### DISCUSSION THREAD

**Discussion by teredactle** [gtxaspec/wz_mini_hacks#328] [Q&A]
So, reading through the notes I see this:

_The RTSP server supports up to two video streams provided by the camera, 1080p/360p **(1296p/480p for the DB3 [DoorBell3])**_

Is this firmware supported on the wired doorbell then???

**Comment by KevinKadow** [comment]
[No MicroSD slot](https://support.wyze.com/hc/en-us/articles/4408652603675-Does-Wyze-Video-Doorbell-Pro-include-a-microSD-card-)

**Reply by shamlord** [reply]
Gotta love their explanation… 

“Someone could steal the camera and then they’d have the sd card too.”

Wouldn’t that apply to all of the cameras? 🧐 (of course it would)

**Reply by Bob-AOL** [reply]
If they were all mounted waist high at your front door, sure. LOL

**Reply by shamlord** [reply]
The doorbell camera would be easy to steal due to its positioning, but so are any of the others. I have never seen anyone mount a Wyze, Ring, Nest, or any other cloud service camera where it is out of reach of an average height adult. I'm sure some people do it, but it's far from common. 

Besides, the doorbell camera shouldn't be your only source of security. It's ultimately a convenience, at least for me. Better than pulling down a bunch of soffit, running an electrical outlet, and then placing a v3 somewhere on my porch, just for me to move a few years from now. The doorbell cam is an easier solution. I still have another camera facing my porch, covering any approach to the doorbell cam.

**Comment by gtxaspec** [comment]
It does, but installation requires either manual flash of the firmware chip via external programmer, or manual installation using the old wyzehacks method.  there is no way other than re-flash of the firmware chip using an external programmer in case of any issues at the moment.

**Reply by shamlord** [reply]
I’m interested in trying this, but so far I’ve not found any more detailed information about what I might find inside the doorbell cam. I’m definitely going with the original doorbell cam since the pro doorbell is apparently quite different from other Wyze products.

I do not have a chip programmer at present, but if you could point me in the right direction as to what I could buy that would be cool. Bonus points if it’s possible to re-flash without desoldering the chip from the board. Also, since the doorbell cams do not have internal storage, is it possible to hack an SD card into it? My goal would be to use NFS to copy the clips to my NAS.

**Comment by mach7** [comment]
@gtxaspec like @shamlord I am also interested in this. Can you share some more details about getting it working with the doorbells?

**Reply by shamlord** [reply]
I looked at this but ultimately didn't try it. The doorbell has a hidden Micro USB port but no SD card slot. As a result the version of iCamera on the doorbell does not attempt to record video to local storage. Perhaps a hacked version of iCamera in WZ_Mini would remedy that, but what would then happen to the doorbell functionality? 

You can set up a custom Linux stack to trick the camera into loading WZ_Mini like a firmware update, or you can get a programmer and manually flash the memory on the camera. The amount of time necessary to learn and try to do either just wasn't worth it to me given how unlikely it was I would end up with a more useful doorbell camera.

I would love for the doorbell to record video to my NFS, not to mention get longer clips and remove the limitation on back-to-back clips that aren't possible without Wyze Cam Plus. I just haven't had the free time necessary for me to learn and tinker with it.

Ultimately I just decided to run the Doorbell as-is. I have all my Wyze v3 cams set up to record to NFS, and the doorbell records it's video to whatever free cloud storage Wyze offers.
